rust/noosphere-core/src/tracing.rs

```diff
@@ -305,23 +309,45 @@ mod inner {
 
         let subscriber = tracing_subscriber::registry().with(env_filter);
 
+        // This isn't a secret value, just where to send the error reports. We can make this configurable in the future if other folks care.
+        let _guard = sentry::init(("https://553fd6eda33842ed9f088d0c16a147f1@o4505393671569408.ingest.sentry.io/4505399702126593", sentry::ClientOptions {
```

Comment:
Is it specific to our account or is it a single endpoint used for all of Sentry? If it is specific to our account, then we should make it an environment variable.

Reply:
Specific. Moved.